Scenes/Command/Command.gd

```diff
@@ -11,7 +11,7 @@ var inputKey
 var isCurrentlyPressed : bool = false
 
 
-func Command(ID : String, inputKey : int, inputType : String):
+func command(ID : String, inputKey : int, inputType : String):
```

Comment:
Thanks Karbb, I was having a hard time with that capital letter and my OCD.

Reply:
I mean: even if the rest of the PR is not working, I'm going to accept this anyway :rofl: